src/toil/lib/aws/__init__.py

```diff
@@ -168,29 +168,25 @@ def file_begins_with(path, prefix):
     except (URLError, socket.timeout, HTTPException):
         return False
 
+
 def running_on_ecs() -> bool:
     """
     Return True if we are currently running on Amazon ECS, and false otherwise.
     """
     # We only care about relatively current ECS
     return 'ECS_CONTAINER_METADATA_URI_V4' in os.environ
 
-def build_tag_dict_from_env(environment: MutableMapping[str, str] = os.environ) -> Dict[str, str]:
-    tags = dict()
-    owner_tag = environment.get('TOIL_OWNER_TAG')
+
+def tags_from_env() -> Dict[str, str]:
+    try:
+        tags = json.loads(os.environ.get('TOIL_AWS_TAGS', '{}'))
+    except json.decoder.JSONDecodeError:
+        logger.error('TOIL_AWS_TAGS must be in JSON format: {"key" : "value", ...}')
+        exit(1)
+
+    # TODO: Remove TOIL_OWNER_TAG and only use TOIL_AWS_TAGS .
```

Comment:
I don't know if we want to do that; it's a lot simpler to set TOIL_OWNER_TAG to your email than it is to construct valid JSON describing doing that.

Than again, the owner tag convention is just a thing we invented for ourselves. But also this is code we control and it can have optimizations that make us happy.